contracts/arbos/builtin/ArbRetryableTx.sol

```diff
@@ -1,41 +1,70 @@
 
 pragma solidity >=0.4.21 <0.7.0;
 
+/**
+* @title precompiled contract in every Arbitrum chain for retryable transaction related data retrieval and interactions. Exists at 0x000000000000000000000000000000000000006E 
+*/
 interface ArbRetryableTx {
-    // Redeem a redeemable tx.
-    // Revert if called by an L2 contract, or if txId does not exist, or if txId reverts.
-    // If this returns, txId has been completed and is no longer available for redemption.
-    // If this reverts, txId is still available for redemption (until it times out or is canceled).
+
+    /**
+    * @notice Redeem a redeemable tx.
+    * Revert if called by an L2 contract, or if txId does not exist, or if txId reverts.
+    * If this returns, txId has been completed and is no longer available for redemption.
+    * If this reverts, txId is still available for redemption (until it times out or is canceled).
+    @param txId unique identifier of retryabale message: keccak256(requestID, uint(0) )
+     */
     function redeem(bytes32 txId) external;
 
-    // Return the minimum lifetime of redeemable txs.
+    /** 
+    * @notice Return the minimum lifetime of redeemable txn.
+    * @return lifetime in seconds
+    */
     function getLifetime() external view returns(uint);
 
-    // Return the timestamp when txId will age out, or zero if txId does not exist.
-    // The timestamp could be in the past, because aged-out txs might not be discarded immediately.
+    /**
+    * @notice Return the timestamp when txId will age out, or zero if txId does not exist.
+    * The timestamp could be in the past, because aged-out txs might not be discarded immediately.
+    * @param txId unique identifier of retryabale message: keccak256(requestID, uint(0) )
```

Comment:
typo: "retryabale"

Reply:
Also, "requestID" will be mysterious here.  Perhaps we shouldn't say here how the unique ID is computed, or perhaps we should explain what "requestID" is.

Reply:
included the full computation, bc why not